docs/openapi.yaml

```diff
@@ -2086,7 +2086,8 @@ components:
           example: '2019-06-01T00:00:00Z'
           description: |-
             The date and time the campaign ends.
-            Needs to be UTC.
+            This is a required field for some campaign types: `PERIOD_CAMPAIGN`, `EVENT_CAMPAIGN` and `ULL_FLEX_CAMPAIGN`.
```

Comment:
This is not correct 😅
eventDate is currently only used by `EVENT_CAMPAIGN`, and is illegal for other types

Reply:
but if you add this type of info, you also need to add that `interval` and `end` is used by FULL_FLEX_CAMPAIGN and that `period`is used by PERIOD_CAMPAIGN, etc

Reply:
Does openApi support inheritance? maybe we can create separate DTOs for the different campaign types and make that work somehow? 🤔

Reply:
Can you suggest the correction so I can commit it? Use the +/- functionality in the comment field.

Reply:
Just try to explain it in text for now, we can see what's possible with OpenAPI later.

Reply:
OK?
```
          description: |-
            The date and time the campaign ends.
            This is a required field for `EVENT_CAMPAIGN`, and is illegal for other types.
            But: If you add this type of info, you also need to add that interval
            and end is used by `FULL_FLEX_CAMPAIGN` and that periodis used by `PERIOD_CAMPAIGN`, etc.
            Musst be UTC.
            ```

Reply:
Just missing a space in: **that periodis used by**